clients/fossid-webapp/src/main/kotlin/VersionedFossIdService21dot2.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.clients.fossid
+
+import org.ossreviewtoolkit.clients.fossid.model.status.Description
+
+class VersionedFossIdService21dot2(
```

Comment:
Regarding naming: The new classes are now named with a specific version, but the meaning is actually "this version or newer". I am just thinking whether it would be clearer to name the most recent class without a version and somehow mark the legacy classes. Not sure what is better.

Reply:
Yes I think about this too. The problem is we have not so much visibility about this...
What we could do, we could mark the legacy classes with the highest supported version. And the current one without ?

The problem I see is the Git history: when a new version is coming, you need to move the current class to a legacy class with version in one commit. Then in another commit you can create a new class for the actual version.
Otherwise you risk of losing Git history for the legacy class.